torch/ao/quantization/fx/prepare.py

```diff
@@ -121,6 +124,16 @@ def _is_activation_post_process_node(node: Node, named_modules: Dict[str, torch.
     return isinstance(node, torch.fx.Node) and node.op == "call_module" and \
         _is_activation_post_process(named_modules[str(node.target)])
 
+def _get_dtype_and_is_dynamic(obs_or_fq_ctr: Optional[Callable]) -> Tuple[Optional[torch.dtype], bool]:
+    """ Given a constructor for observer or fake quant module, returns
+    a Tuple of dtype and is_dynamic
+    """
+    if obs_or_fq_ctr is None:
+        return None, False
+    else:
+        obs_or_fq = obs_or_fq_ctr()
```

Comment:
fyi: I think you can use `inspect.signature` to get dtype from the constructor args if you want to avoid instantiating.

Reply:
thanks, there are other occurrences, will leave this as a future cleanup